File: generator/output/ts-model/user.model.ts
class UserModel {
  public id: number = 0;

  // deleted 是否删除
  public deleted?: number = 0;

  // email 邮箱地址
  public email?: string;

  // mobile 手机号
  public mobile: string;

  // password 密码
  public password: string;

  // domain 用户域名
  public domain?: string;

  // active 0未激活，1激活
  public active: number = 0;

  // username 用户名
  public username: string;

  public created_at: string;

  public updated_at: string;
}

const UserModelFields = {
  id: "id",
  deleted: "deleted",
  email: "email",
  mobile: "mobile",
  password: "password",
  domain: "domain",
  active: "active",
  username: "username",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {UserModel, UserModelFields};

File: backend/src/main/java/com/licenseair/backend/domainModel/UserModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* User
*/
@Data()
public class UserModel {
  public Long id = null;

  /**
  * 是否删除
  */
  public boolean deleted = false;

  /**
  * 邮箱地址
  */
  public String email = null;

  /**
  * 手机号
  */
  public String mobile = null;

  /**
  * 密码
  */
  public String password = null;

  /**
  * 用户域名
  */
  public String domain = null;

  /**
  * 0未激活，1激活
  */
  public Integer active = null;

  /**
  * 用户名
  */
  public String username = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
